src/main/java/uk/gov/hmcts/reform/sscs/service/ccdupdate/CcdStateUpdateService.java

```diff
@@ -28,7 +27,7 @@ public class CcdStateUpdateService {
     public void updateListed(HearingGetResponse hearingResponse, @Valid SscsCaseData sscsCaseData)
             throws UpdateCaseException {
 
-        if (!isHearingListingStatusFixed(hearingResponse)) {
+        if (isHearingListingNotStatusFixed(hearingResponse)) {
```

Comment:
minor/optional: would change the wording of this to `hearingListingIsNotStatusFixed` as it reads better imo / same goes for the other is/Not methods.